ktor-client/ktor-client-features/ktor-client-json/ktor-client-jackson/jvm/src/io/ktor/client/features/json/JacksonSerializer.kt

```diff
@@ -11,9 +11,17 @@ import io.ktor.http.*
 import io.ktor.http.content.*
 import kotlinx.io.core.*
 
-class JacksonSerializer(block: ObjectMapper.() -> Unit = {}) : JsonSerializer {
+class JacksonSerializer : JsonSerializer {
 
-    private val backend = jacksonObjectMapper().apply(block)
+    private val backend: ObjectMapper
+
+    constructor(block: ObjectMapper.() -> Unit={}) {
+        backend = jacksonObjectMapper().apply(block)
+    }
+
+    constructor(mapper: ObjectMapper) {
```

Comment:
It looks like it should become primary, isn't it?

```
class JacksonSerializer(private val backend: ObjectMapper) {
    constructor(block: ObjectMapper.() -> Unit = {}) : this(jacksonObjectMapper().apply(block))
```
```

Reply:
Yes, you are right!) Changed it.